src/community/advanced-data-card.tsx

```diff
@@ -200,8 +226,10 @@ type AllowedExtra =
 
 type TextAs = 'h1' | 'h2' | 'h3' | 'h4';
 
-interface AdvancedDataCardProps {
+type AdvancedDataCardProps = {
     onPress?: () => void;
+    href?: string | undefined;
+    to?: undefined;
```

Comment:
remove to/href/onPress from common props, you are already defining them in the ExclusifyUnion